snuba/web/delete_query.py

```diff
@@ -0,0 +1,36 @@
+from snuba.clickhouse.formatter.query import format_query
+from snuba.clickhouse.query import Query
+from snuba.datasets.storages.storage_key import StorageKey
+from snuba.query.conditions import combine_and_conditions
+from snuba.query.data_source.simple import Table
+from snuba.query.dsl import column, equals, in_cond, literal, literals_tuple
+
+
+def construct_condition(body):
```

Comment:
This currently will not handle the case of `gte` or `lte` conditions, like we would do with timestamp.